src/sagemaker/pytorch/model.py

```diff
@@ -137,34 +136,21 @@ def prepare_container_def(self, instance_type, accelerator_type=None):
                 For example, 'ml.p2.xlarge'.
             accelerator_type (str): The Elastic Inference accelerator type to
                 deploy to the instance for loading and making inferences to the
-                model. For example, 'ml.eia1.medium'.
+                model. Currently unsupported with PyTorch.
 
         Returns:
             dict[str, str]: A container definition object usable with the
             CreateModel API.
         """
-        lowest_mms_version = packaging.version.Version(self._LOWEST_MMS_VERSION)
-        framework_version = packaging.version.Version(self.framework_version)
-        is_mms_version = framework_version >= lowest_mms_version
-
         deploy_image = self.image
         if not deploy_image:
             region_name = self.sagemaker_session.boto_session.region_name
-
-            framework_name = self.__framework_name__
-            if is_mms_version:
-                framework_name += "-serving"
-
-            deploy_image = create_image_uri(
-                region_name,
-                framework_name,
-                instance_type,
-                self.framework_version,
-                self.py_version,
-                accelerator_type=accelerator_type,
+            deploy_image = self.serving_image_uri(
+                region_name, instance_type, accelerator_type=accelerator_type
```

Comment:
[Minor] Keyword/Named arguments make kittens happy.

Reply:
talked offline - I don't see a significant benefit of doing `region_name=region_name` over `region_name`